static/js/list_widget.js

```diff
@@ -293,7 +293,10 @@ export function create($container, list, opts) {
     // the sorting function is either the function or string that calls the
     // function to sort the list by. The prop is used for generic functions
     // that can be called to sort with a particular prop.
-    widget.set_sorting_function = function (sorting_function, prop) {
+    widget.set_sorting_function = function (sorting_function, prop, reverse_mode) {
+        if (reverse_mode) {
+            widget.set_reverse_mode(reverse_mode);
+        }
```

Comment:
I don't like the name `reverse_mode` here; I think we want something like `initially_descending` as the label; but I also think it might be cleaner to be a separate property than the `init_sort` property so that it can be named rather than an unexplained extra `true` parameter.